### ISSUE TITLE
providers request

### ISSUE THREAD

**Issue by alegye** [StreamerApp/Streamer#5]
https://vumoo.to
https://primewire 
https://moviestars.to 
https://123chill.to 
https://vwv.ssoap2day.to
https://watchmovie.cc
https://www2.kiss-cartoon.io
https://www.kissmovies.io
http://ww2.kisscartoon.gr
https://kissanime.pe[
https://wwatchseries.com
https://www.primewire.li
https://kissasian.pe
https://hdfull.fm
https://www.wcostream.com
https://tvshows88.com
https://theflix.to
https://lookmovie.io

please thanks you 
hopefully they can implement these pages to streamer

**Comment by alegye** [comment]
https://www.kissmovies.io 
https://www2.kiss-cartoon.io 
https://www.wcostream.com 

it can be implemented is 3 pages are few

**Comment by alegye** [comment]
they are going to add more scrapers and more providers of movies and series

**Comment by sasagr** [comment]
I checked https://tvshows88.com/ and it has a rich library of movies and tvshows and the recent tvshows are also offering 1080p resolution that I m not always seeing in flixtor for example. They also provide subtitles.
I think it could be good to add it if possible. Pls @streamer-bot let us know

**Comment by sasagr** [comment]
@streamer-bot I checked and you are right!
I checked the tv shows, 1883 s01e07 and they are in both coming from jevq.mcloud.to and there is the 1080p resolution.
But I have the question... is the highest resolution scraped by default in the apple tv, because I have the feeling that I m getting maybe the 720p. Is there something we can have to be sure f what we re getting. I see only HD written on screen.

**Comment by alegye** [comment]
hello friend

**Comment by alegye** [comment]
hi

**Comment by alegye** [comment]
hi friends, any lucky with the implementation?

**Comment by alegye** [comment]
Hello you can implement the 5 pages friends

sorry is that I inadvertently closed my issue and put it back open

**Comment by alegye** [comment]
please tmdb movies tvshows
please imdb movies tvshows


It can be implemented to watch movies and series

**Comment by alegye** [comment]
https://www.kissmovies.io/ 
https://www2.kiss-cartoon.io/ 
https://www.wcostream.com/ 
 https://kissanime.tube/


you can check if you can still implement these pages

**Comment by Raviolii** [comment]
Can you German Movies and series like:

https://bs.to
https://streamcloud.cx
https://hdfilme.cx
https://kinokiste.club

**Comment by alegye** [comment]
Hello

**Comment by shuaiwang88** [comment]
ww5.0123movie.net
watchserieshd.bz
are stable

**Comment by imabhaysutar** [comment]
https://github.com/LagradOst/CloudStream-3 
what he has done is there are no hardcoded links, the app pulls latest active links for said provider from [https://raw.githubusercontent.com/LagradOst/CloudStream-3/master/providers.json](JSON) file here.. that way it is actively being updated without needing to update app just to fix this.

**Comment by imabhaysutar** [comment]
Yes, another benefit is that most provider domains get changed often, so this would also help that situation. Cheers!

**Comment by madrid2022ok** [comment]
https://movidy.pro/
https://olgply.com/
https://hydrawire.tv/
https://azmovies.app/
https://www.goojara.to/

**Comment by iMonZ** [comment]
> Can you German Movies and series like:
> 
> https://bs.to https://streamcloud.cx https://hdfilme.cx https://kinokiste.club

Kinox would be great as well!
